python/cudf/cudf/tests/test_extension_compilation.py

```diff
@@ -187,18 +196,20 @@ def test_is_na(fn):
 
     device_fn = cuda.jit(device=True)(fn)
 
-    @cuda.jit(debug=True)
-    def test_kernel():
+    @cuda.jit
+    def test_kernel(err):
         valid_is_na = device_fn(valid)
         invalid_is_na = device_fn(invalid)
 
         if valid_is_na:
-            raise RuntimeError("Valid masked value is NA and should not be")
+            err[0] = 1
 
         if not invalid_is_na:
-            raise RuntimeError("Invalid masked value is not NA and should be")
+            err[0] = 2
 
+    err = cp.asarray([0], dtype="int8")
     test_kernel[1, 1]()
```

Comment:
To fix the issue above:

```suggestion
    test_kernel[1, 1](err)
```

Reply:
Yup, that fixed it. Thanks @gmarkall ! 

@shwina FYI I'm committing this change, tested this locally.